lib/govuk_docker/doctor/checkup.rb

```diff
@@ -87,7 +87,7 @@ def running_user_message
     end
 
     def dnsmasq_resolver?
-      File.read('/etc/resolver/dev.gov.uk').strip == "nameserver 127.0.0.1\nport 53"
+      File.read('/etc/resolver/dev.gov.uk') == File.read(File.join(GovukDocker::Paths::govuk_docker_dir, 'config', 'dnsmasq.conf'))
```

Comment:
It might be worth extracting `File.join(GovukDocker::Paths::govuk_docker_dir, 'config', 'dnsmasq.conf'` into a variable or method here as this line is quite long.

Reply:
Have created a `relative_to_govuk_docker` method in `GovukDocker::Paths`.